include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -267,24 +255,24 @@ ORT_API(int, OrtSetSessionThreadPoolSize, _In_ OrtSessionOptions* options, int s
   */
 ORT_API(void, OrtSessionOptionsAppendExecutionProvider, _In_ OrtSessionOptions* options, _In_ OrtProviderFactoryInterface** f);
 
-ORT_API(void, OrtAddCustomOp, _In_ OrtSessionOptions* options, const char* custom_op_path);
+ORT_API(void, OrtAppendCustomOpLibPath, _In_ OrtSessionOptions* options, const char* lib_path);
```

Comment:
Why do you call it append?

Reply:
Yuan thought this was much clearer since it's appending it to the end of the Custom op libraries. It's not 'adding it' to them (which implies there might be some sorting done)